README.md

```diff
@@ -318,5 +320,28 @@ If you've completed all of the above steps, run `bx app push`, and watch the out
 
 # References
 [LoopBack](https://loopback.io)
+
+# Privacy Notice
+
+Sample web applications that include this package may be configured to track deployments to [IBM Bluemix](https://www.bluemix.net/) and other Cloud Foundry platforms. The following information is sent to a [Deployment Tracker](https://github.com/IBM/metrics-collector-service) service on each deployment:
```

Comment:
We need to start calling it IBM Cloud now, right?

Reply:
Oh you're right. 👍